airflow/models/dagrun.py

```diff
@@ -43,6 +43,20 @@
 from airflow.utils.types import DagRunType
 
 
+class _TISchedulingDecision(NamedTuple):
```

Comment:
```suggestion
class TISchedulingDecision(NamedTuple):
```

I don't know why I made this "private" -- and it looks a bit odd to use it in a return value as it was